hack/lib/init.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
```

Comment:
There's already a init.sh file in hack directory. Please merge the contents of hack/init.sh into hack/lib/init.sh and delete hack/init.sh. After this, modify hack/verify-gofmt.sh, hack/verify-golint.sh and hack/verify-govet.sh to source hack/lib/init.sh